templates/default/packer_pipeline.config.xml.erb

```diff
@@ -21,6 +21,10 @@
         </hudson.model.StringParameterDefinition>
       </parameterDefinitions>
     </hudson.model.ParametersDefinitionProperty>
+    <jenkins.branch.RateLimitBranchProperty_-JobPropertyImpl plugin="branch-api@2.0.8">
```

Comment:
Is this plugin already being installed?

Reply:
The branch plugin is installed [here](https://github.com/osuosl-cookbooks/osl-jenkins/blob/master/attributes/plugins.rb#L17).